hugo/content/docs/extend-your-key-from-gpg.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Extend your PGP key from GPG with Fluidkeys"
+weight: 4
+---
+
+# How to extend your PGP key expiry date with Fluidkeys (and never have to do it again)
+
+Fluidkeys is a command line application that makes it simple for engineering teams to get up and
+running with PGP.
+
+You can use it to automatically maintain a key you've already made using GnuPG, send end-to-end encrypted secrets to other and use `pass` the open-source unix password manager to manage your team passwords.
+
+Contents:
+
+1. [Prerequisites](#prerequisites)
+2. [Import your PGP key into Fluidkeys](#import-key)
+3. [Push it to the keyservers](#push-to-keyservers)
```

Comment:
this list doesn't reflect the sections in the doc! (there's a Fluidkeys server section)